docs/en/docs/sql-manual/sql-functions/aggregate-functions/group_uniq_array.md

```diff
@@ -0,0 +1,69 @@
+---
+{
+    "title": "GROUP_UNIQ_ARRAY",
+    "language": "en"
+}
+---
+
+<!-- 
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+## GROUP_UNIQ_ARRAY
+### description
+#### Syntax
+
+`ARRAY<T> collect_set(expr[,max_size])`
+
+Creates an array from different argument values,with the optional max_size parameter limits the size of the resulting array to `max_size` elements.
+
```

Comment:
we need to add more detailed instructions here and the examples below.

Reply:
zh-CN versioon too.

Reply:
Ok,I will specify them later.